Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs

```diff
@@ -1827,5 +1827,34 @@ public async Task<IEnumerable<Share>> ShareSight(long? sightId, IEnumerable<Shar
         }
         
         #endregion
+
+        #region Cell History
+
+        public async Task<IEnumerable<CellHistory>> GetCellHistory(long? sheetId, long? rowId, long? columnId, string accessToken = null)
+        {
+            if (sheetId == null)
+            {
+                throw new Exception("Sheet ID cannot be null");
+            }
+
+            if (rowId == null)
+            {
+                throw new Exception("Row ID cannot be null");
+            }
+
+            if (columnId == null)
+            {
+                throw new Exception("Column ID cannot be null");
+            }
+
+            var response = await this.ExecuteRequest<IndexResultResponse<CellHistory>, CellHistory>(HttpVerb.GET,
+                string.Format("sheets/{0}/rows/{1}/columns/{2}/history?include", sheetId, rowId, columnId), null,
```

Comment:
You have the `include` keyword in the URL but no include parameter options. There's a couple of routes you can go here:
1. Drop the include altogether
2. Keep the include but make it a manually entered list of strings that you join() into a comma-separated list
3. Add each of the include flags as an optional boolean argument to the method.
4. (Probably the actual right way) Define an enum for the cell/cell history object include flags type and accept an iEnumerable of the enum type as an argument.

Many of the other methods on the client object go with option 3, but that's not necessarily the best way (or my preferred way) of doing it. I'd probably just recommend option 1 for now unless you've got time to kill and feel like adding a better implementation. Option 2 is ok but lazy (though it has the benefit of not requiring an update should the API implement additional include flags in the future). Option 4 is the 'right' way.

Reply:
Updated per method 4.